etl/steps/data/garden/emdat/2025-10-29/natural_disasters.meta.yml

```diff
@@ -106,6 +106,23 @@ definitions:
     - *definition-disaster
     - *definition-dead
     - *definition-unknown-deaths
+  # Metadata for variables adjusted for inflation using the GDP deflator.
+  description_short_constant_usd: &description-short-constant-usd |-
+    This data is expressed in US dollars. It is adjusted for inflation but does not account for differences in living costs between countries.
+  footnote_constant_usd: &footnote-constant-usd |-
+    This data is expressed in constant {BASE_YEAR} US$.
+  description_key_inflation_adjustment: &description-key-inflation-adjustment
+    - This variable has been adjusted for inflation using the OWID Deflator dataset (Our World in Data, 2025).
```

Comment:
Looks like you're no longer using this dataset. Also, it's weird to express it like that for external readers. They don't have any reference about it.